crates/next-api/src/module_graph.rs

```diff
@@ -521,9 +526,10 @@ async fn validate_pages_css_imports(
         }
 
         // We allow imports of global CSS files which are inside of `node_modules`.
-        let module_name_contains_node_modules = module_name_map
+        let module_name_contains_node_modules = module_ident_map
```

Comment:
is this the only thing we need from the idents?

can module_ident_map instead be a simple Set of node_modules modules and be built by just quering the path?  that would be smaller and cheaper to construct

Reply:
Is it easier to construct? we are bound by 
```
        .node_weights()
        .map(async |n| Ok((n.module(), n.module().ident().to_string().await?)))
        .try_join()
```
anyway. I could either store that in a hashmap or a filtermap it down again into a hashset
but either way, it's only stored for the duration of the traversal.

Reply:
i figure it is kind of wasteful to construct the full string when we really just need to check for `/node_modules/` in the filepath

Ok(n.module(), n.module().ident().path().await?.path.contains("/node_modules/")

The ident tostrings are probably cached at this point so the win would mostly about constructing a smaller datastructure  (assuming we can `filter_map` i guess)